Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.azsli

```diff
@@ -145,15 +145,15 @@ option enum class OpacityMode {Opaque, Cutout, Blended, TintedTransparent} o_opa
         float4x4 objectToWorld = ObjectSrg::GetWorldMatrix();
         float3x3 objectToWorldIT = ObjectSrg::GetWorldMatrixInverseTranspose();
 
-        float3 vertexNormal, vertexTangent, vertexBitangent;
```

Comment:
nit: I would consider renaming this to vertexNormalWS to help delineate a vertex normal in world space vs. the surface normal.